config/owasp/suppressions.xml

```diff
@@ -1,42 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress>
-    <notes><![CDATA[name: spring-plugin-core and spring-plugin-metadata 2.0.0.RELEASE]]></notes>
-    <cve>CVE-2013-4152</cve>
-    <cve>CVE-2013-7315</cve>
-    <cve>CVE-2014-0054</cve>
-    <cve>CVE-2016-1000027</cve>
-    <cve>CVE-2022-22965</cve>
-    <cve>CVE-2022-22968</cve>
-    <cve>CVE-2022-22976</cve>
-    <cve>CVE-2022-22978</cve>
-  </suppress>
   <suppress>
     <notes><![CDATA[name: spring-boot 3.0.6]]></notes>
```

Comment:
I guess, you could've bumped spring-boot to 3.0.8 and this suppression would not be needed. Leaving fairly empty suppression file :)